.github/workflows/app_build.yml

```diff
@@ -0,0 +1,304 @@
+name: C/C++ CI for OpenStudioApplication
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Suggest running CI on pull requests to master or develop and on push to master

Reply:
These might be different, we might want to make tags on push to master

Reply:
Done, we can adjust later.